pkg/kubectl/cmd/resource/get.go

```diff
@@ -662,43 +664,44 @@ func cmdSpecifiesOutputFmt(cmd *cobra.Command) bool {
 
 // outputOptsForMappingFromOpenAPI looks for the output format metatadata in the
 // openapi schema and returns the output options for the mapping if found.
```

Comment:
Godoc should say "modifies the passed print options" and the name of the method should reflect it - `updatePrintOptionsForOpenAPI`